web/themes/webspark/renovation/src/components/accordions/accordion.twig

```diff
@@ -46,7 +46,7 @@
     {% else %}
        class="collapse"
     {% endif %}
-     aria-labelledby="{{ heading }}"
+     aria-labelledby="accordion-header-{{ link_id }}"
```

Comment:
Why didn't you remove the `aria-labelledby` as recommended on the ticket?

Reply:
Hi @mlsamuelson 
You're right, I just based on the image attached to the description in order to implement this change. Then, I realized that I need to remove it. 
This is done. Thanks.